consensus/src/consensus_provider.rs

```diff
@@ -39,6 +39,8 @@ pub fn start_consensus(
         .enable_all()
         .build()
         .expect("Failed to create Tokio runtime!");
+    // REVIEW: Should MempoolProxy etc be the object passed to this function (vs passing a channel
```

Comment:
we're writing rust, use channel! - I think mempool is running in a different runtime not like libradb which is shared via Arc